handsontable/src/core.js

```diff
@@ -2636,38 +2636,69 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
   };
 
   /**
-   * Allows altering the table structure by either inserting/removing rows or columns.
-   * This method works with an array data structure only.
+   * The `alter()` method lets you alter the grid's structure
+   * by adding or removing rows and columns at specified positions.
+   *
+   * ::: tip
+   * The `alter()` method works only when your [`data`](@/api/options.md#data)
+   * is an [array of arrays](@/guides/getting-started/binding-to-data.md#array-of-arrays).
+   * :::
+   *
+   * ```js
+   * // above row 10 (by visual index), insert 1 new row
+   * hot.alter('insert_row_above', 10);
+   * ```
+   *
+   *  | Action               | Description |
+   *  | -------------------- | ----------- |
+   *  | `'insert_row_above'` | Insert rows above a specified row. |
+   *  | `'insert_row_below'` | Insert rows below a specified row. |
+   *  | `'remove_row'`       | Remove specified rows. |
+   *  | `'insert_col_start'` | Depending on your [`layoutDirection`](@/api/options.md#layoutdirection), insert columns to the left (in LTR) or to the right (in RTL) of a specified column. |
+   *  | `'insert_col_end'`   | Depending on your [`layoutDirection`](@/api/options.md#layoutdirection), insert columns to the right (in LTR) or to the left (in RTL) of a specified column. |
+   *  | `'remove_col'`       | Remove specified columns. |
+   *  | `'insert_row'`       | (<b>Deprecated</b>) An alias for `'insert_row_above'`. |
+   *  | `'insert_col'`       | (<b>Deprecated</b>) An alias for `'insert_col_start'`. |
```

Comment:
It is not true that `insert_row` and `insert_col` are just aliases. It might cause a bug if someone follows this advice and simply renames the parameters in their code. 

As explained on https://github.com/handsontable/handsontable/pull/9955, `insert_row` works as  `insert_row_above` or `insert_row_below`, depending on the row index. The same about `instert_col`.

Reply:
@warpech – done: [b2bd1aa](https://github.com/handsontable/handsontable/pull/10046/commits/b2bd1aa95c302d3872fe11b004a02505f1acb7a7)

Reply:
Thanks, I have a comment to that but I will add it as a separate thread